server/src/api/v1.py

```diff
@@ -766,6 +796,10 @@ def generate_token(allowed_resources, secret_key):
         token_payload["max_priority"] = allowed_resources["max_priority"]
     if "allowed_queues" in allowed_resources:
         token_payload["allowed_queues"] = allowed_resources["allowed_queues"]
+    if "max_reservation_time" in allowed_resources:
+        token_payload["max_reservation_time"] = allowed_resources[
```

Comment:
I think it would be extremely useful if there was some documentation around the structure and contents of these "client permissions" or "allowed_resources". A schema, a reference or even an example somewhere would really really help. Also, I am wondering if using a different name in the database (client permissions) and the code (allowed resources) was deliberate? Aren't these two referring to the same concept?

Reply:
Perhaps in the docstring of this function we could add a reference like "see `retrieve_token` for more information on the contents of the token payload".